### ISSUE TITLE
A problem with upgradeElement

### ISSUE THREAD

**Issue by Graf009** [tleunen/react-mdl#13]
I am using the react-model and react-mdl have an error when called 
```javascript
upgradeElement(React.findDOMNode(this))
```
```javascript
TypeError: Cannot read property 'firstChild' of undefined
    at Object.ReactMount.findComponentRoot
```
It may be replaced upgradeElement(React.findDOMNode(this)) by upgradeDom(), this solves the problem. What are some other ideas?

**Comment by tleunen** [comment]
Hmm... Do you have a code example?
BEcause `upgradeDom` uses `upgradeElement` internally... So I'm a bit surprised if the error only occurs when using `upgradeElement`.

In the end, it might be related to this other issue https://github.com/tleunen/react-mdl/issues/4. But I'm not sure it was the exact same error. With a code example, I could investigate this.

**Comment by Graf009** [comment]
I'll add an example later

**Comment by Graf009** [comment]
Demo: https://gist.github.com/Graf009/ad0131583c2bb9a07f7c
I think the problem in React.findDOMNode(this), React can not find the element node.

**Comment by tleunen** [comment]
Would need to investigate on this one.
Probably caused by the way `react-modal` works.

**Comment by tleunen** [comment]
Might be fixed by #48. Please enter a new issue if your problem persists (and which component is problematic if possible)
